lambdas/get-addresses/package.json

```diff
@@ -1,30 +1,34 @@
 {
-    "name": "hello_world",
+    "name": "get_addresses",
     "version": "1.0.0",
-    "description": "hello world sample for NodeJS",
+    "description": "GetAddresses Typescript Node Lambda",
     "main": "app.js",
     "dependencies": {
-        "esbuild": "0.14.14"
+        "esbuild": "0.14.14",
+        "@aws-sdk/client-dynamodb": "3.150.0",
+        "@aws-sdk/lib-dynamodb": "3.150.0"
```

Comment:
esbuild doesnt deploy unless it knows about these types in the dependancies... Its strange as I would assume esbuild is aware that lambda knows that we have the entire SDK